lib/ohai/plugins/linux/platform.rb

```diff
@@ -113,6 +129,15 @@ def os_release_file_is_cisco?
       # no way to determine platform_version in a rolling release distribution
       # kernel release will be used - ex. 3.13
       platform_version `uname -r`.strip
+    elsif (os_release_info = os_release_file_is_cisco?)
+      # Cisco platform not based on known distro above
+      if os_release_info['ID'] == 'nexus'
+        platform 'nexus'
+        platform_family 'cisco-wrlinux'
+      elsif os_release_info['ID_LIKE'] =~ /cisco-wrlinux/
```

Comment:
This is for cisco-wrlinux platforms that aren't nexus. We have at least one coming down the pipe that I know of, but I'm not sure what its platform designator will be yet. Personally I'd prefer to just set `platform = os_release_info['ID']` so as to be future-proof, but this seemed to be discouraged - see #572.